src/dex/balancer-v2/balancer-v2-pool.ts

```diff
@@ -1,744 +0,0 @@
-import { Interface } from '@ethersproject/abi';
```

Comment:
So all this logic from this file has just been refactored into separate files under "pools" folder?
It is hard to determine what was modified vs just moved.

Reply:
@skypper yes the files was moved from ../balancer-v2/ to ../balancer-v2/pools/